app/assets/stylesheets/components/page_specific/home_beta/_tool_promo.scss

```diff
@@ -0,0 +1,44 @@
+.tool_promo-beta {
+  @include column(12);
+  border: 0;
+
+  @include respond-to($mq-m) {
+    @include column(6);
+  }
+
+  @extend .callout;
+  @extend .callout--tool;
+}
+
+.tool_promo-beta__item {
+  @include body(14, 24);
+  background-color: $color-grey-ten;
+  border-bottom: 2px solid $color-orange-medium;
+  padding: $baseline-unit*2;
+  color: black;
+  position: relative;
+}
```

Comment:
This element could usefully have more padding on the right hand side to avoid overlapping at some small widths.

![image](https://user-images.githubusercontent.com/6080548/34873163-f34d944a-f78b-11e7-9080-962adc3c3069.png)

Reply:
@davidtrussler Thanks, please pull the latest I've made these changes

Reply:
Yes, looks good!